nautobot/core/templatetags/helpers.py

```diff
@@ -870,9 +870,7 @@ def saved_view_modal(
 ):
     from nautobot.extras.forms import SavedViewModalForm
     from nautobot.extras.models import SavedView
-
-    param_dict = {}
-    filters_applied = parse_qs(params)
```

Comment:
Sorry for changing the direction, but I think this will be a more correct and generic fix. So the problem originates from parse_qs which returns every query value as a list for example `{"hidden": [True]}`, which caused the error we saw in the issue. The more generic fix should be using `is_single_choice_field` to determine whether to pass the value as a list or a single value into the Saved View config.

Reply:
Never apologize for making things more correct! :-)